tests/integration/test.replication.js

```diff
@@ -62,9 +62,18 @@ adapters.forEach(function (adapters) {
 
     function verifyInfo(info, expected) {
       if (!testUtils.isCouchMaster()) {
-        info.update_seq.should.equal(expected.update_seq, 'update_seq');
+        if (typeof info.doc_count === 'undefined') { 
+          // info is from Sync Gateway, which allocates an extra seqnum
+          // for user access control purposes.
```

Comment:
Duck typing is fine, but the comments are much appreciated. I worry the tests may end up getting pretty gnarly if we have a lot of special logic for CSG vs Couch master vs Couch 1.x vs Pouch Server, but so far it hasn't been too bad.